src/types/types.ts

```diff
@@ -0,0 +1,11 @@
+interface NavbarLinkType {
+  label: string;
+  href: string;
+}
+
+interface ShowcaseDetailsType {
+  nameOrRole: string;
+  summary: string;
+}
```

Comment:
Nit: How about this?

```suggestion
interface NavbarLink {
  label: string;
  href: string;
}

interface ShowcaseDetails {
 title: string;
  summary: string;
}
```

Reply:
Yes, it looks cleaner with this naming convention.

I have updated `interface` names: (https://github.com/krushna-simform/03-portfolio/pull/2/commits/1dfdf002d9a9358c6d236d9ac03be6cf5fa062ef)
- `NavbarLinkType` -> `NavbarLink`
- `ShowcaseDetailsType` -> `ShowcaseDetails`